example/interface.js

```diff
@@ -8,7 +8,7 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
-import { Vircadia, DomainServer, AudioMixer, AvatarMixer, MessageMixer, Uuid } from "../dist/Vircadia.js";
+import { Vircadia, DomainServer, AudioMixer, AvatarMixer, MessageMixer, Uuid, EntityServer } from "../dist/Vircadia.js";
```

Comment:
```suggestion
import { Vircadia, DomainServer, AudioMixer, AvatarMixer, MessageMixer EntityServer, Uuid } from "../dist/Vircadia.js";
```
Primary items before secondary (i.e., Uuid).

Reply:
Fixed.